### DISCUSSION TITLE
List people

### DISCUSSION THREAD

**Discussion by gozeloglu** [community/community#8647] [Other Feature Feedback, Questions, & Ideas] [answered]
Please list people that are watching, forked, and starred the repository as before. I am not able to see who are starred, forked, or watched the repositories even my own repository. It was a nice feature, I did not understand why it was removed in this feature.

**Comment by llrs** [comment, marked as answer]
This has been moved to the code [tab of a repository](https://github.com/github/docs) on the right side together with license and other information:
![Screenshot from 2021-12-10 11-53-18](https://user-images.githubusercontent.com/6818218/145562849-273af8a0-ac19-4fd3-b864-9474a471bbde.png)

**Comment by gozeloglu** [comment]
Thank you @llrs. I did not realize that.
